etc/workflows/failedRecording.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>failedrecording</id>
+  <title>Recording failed</title>
+  <tags>
+  <tag>editor</tag>
+  <tag>archive</tag>
+  </tags>
+  <description/>
```

Comment:
Can you add a description so that people understand for what this workflow is meant to be used?

Reply:
Done, good point.